src/main/resources/com/google/api/codegen/ruby/method_sample.snip

```diff
@@ -0,0 +1,134 @@
+@extends "ruby/common.snip"
+
+@snippet sampleCode(apiMethod)
+    @@example
+    @let lines = util.getDocLines(sampleCode_(apiMethod))
+        @join line : lines
+            {@""}  {@line}
+        @end
+    @end
+@end
+
+@snippet sampleCode_(apiMethod)
+    require "{@apiMethod.initCode.apiFileName}"
+
+    {@apiMethod.apiClassName} = {@apiMethod.initCode.packageName}\
+        ::{@apiMethod.apiClassName}
```

Comment:
This line should be abstracted as an import, then it can be included with the rest of the imports.